### PR TITLE
Feature/add dbiz

### PR DESCRIPTION
**Pull request by dromer**
not functional yet, needs work for dark panels and resolving of font licensing

### CODE REVIEW

File: docs/LICENSES.md [hunk 1]
```diff
@@ -106,6 +107,12 @@ Below is a list of artwork licenses from plugins
 | cf/VT323-Regular.ttf                    | OFL-1.1-no-RFN   | |
 | ChowDSP/*                               | GPL-3.0-or-later | Same license as source code |
 | ChowDSP/fonts/RobotoCondensed-*.ttf     | Apache-2.0       | |
+| dBiz/Dark/*                             | CC BY-NC-ND 4.0  | |
```
Comment (falkTX) [inline docs/LICENSES.md:110]: no spaces on SPDX identifiers. use `CC-BY-NC-ND-4.0`

**Comment by falkTX** [comment]
dotmatrix font comes from https://dionaea.com/information/fonts.php which says the font is "freeware".
this is not a valid license, it is questionable if CC-BY and freeware are allowed to even mix.
it must be replaced for use in cardinal

**Comment by falkTX** [comment]
The other font has details on https://www.dafont.com/rounded-elegance.font
Seems to be a fix of 3 other fonts - "Giorgio", "Abeat By Ky" and "Ukiji"

On the [comments](https://www.dafont.com/font-comment.php?file=rounded_elegance) the author says:

> I inform everyone that this source is free for any type of use, whether private, public or commercial. Make a good use.

This is not a legally valid license. We need to check the other 3 fonts it is based on.

**Comment by dBiz** [comment]
Hi, if needed I can change the font for adding my modules to Cardinal, I'm not a coder but let me know if I can help in any way ;)

**Comment by dromer** [comment]
@dBiz cheers!
First would be to check if any of these included ones are your fancy -> https://github.com/VCVRack/Rack/tree/5551617afff182925940908eaf73a7d7361303cc/res/fonts

Other inspiration would be other module designs -> https://github.com/DISTRHO/Cardinal/blob/952df617e838a0d862678adef2a1dd170e56b222/docs/LICENSES.md#plugins

But we are still looking to maybe create a "font pool" from all the common ones, so we can minimize the number of static files we need to ship.

**Comment by dBiz** [comment]
Yep no prob, I'll replace the font with a Rack/res/font one ;)

**Comment by dromer** [comment]
Ok, managed to resolve all conflicting module/type names and set darkpanel when loading the modules.
But the panel switch needs some more work (library looks dark, and we want to be able to switch to light-mode as well).

Should also remove the menu item on setting `dark as default` for cardinal builds. Might need a patch?

Lets see what @falkTX thinks ;)

**Comment by dBiz** [comment]
I'm open to any changes/suggestion/pull ;) my code skill needs for sure an improvement

**Comment by dBiz** [comment]
I've update my pack with minor bugs and now seems to work as dark in browser, hope it helps ;)

**Comment by dromer** [comment]
@dBiz nice! Seems to work well :)

**Comment by falkTX** [comment]
merged as 46dcb39eddebdbe5f85ab35aca23d15835c0fffe

**Pull request closed without merging**